module/Photo/src/Photo/Controller/PhotoController.php

```diff
@@ -10,17 +10,21 @@ class PhotoController extends AbstractActionController
 
     public function indexAction()
     {
-        $albums = $this->getAlbumService()->getAlbums();
         //add any other special behavior which is required for the main photo page here later
-
+        $years = $this->getAlbumService()->getAlbumYears();
+        $year = $this->params()->fromRoute('year');
+        // If no year is supplied, use the current year.
+        if(is_null($year)) $year = (date("m") < 7) ? date("Y") - 1 : date("Y");
```

Comment:
De 7 klopt, want het is andersom; deze statement wordt gebruikt om het huidige verenigings jaar te bepalen. Bijvoorbeeld:

Als het 1-7-2015 is, dan date("m") < 7 == false, dus het verenigings jaar is 2015/2016
Als het 30-6-2015 is, dan date("m") < 7 == true, dus het verenigings jaar is 2014/2015

Dit heeft inderdaad betere documentatie nodig en de andere dingen die je noemde zal ik ook fixen.
.